components/ai_chat/resources/page/routes.tsx

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import { useNavigation, useParams } from '$web-common/navigation/Context'
+import { useEffect } from 'react'
+
+export function useSelectedConversation() {
+    return useParams().chatId
+}
+
+const routes = [
+    '/',
+    '/{chatId}'
+]
+
+export function Routes() {
```

Comment:
It's not clear to me why this is a react component - could it be just a hook (since all it does it call hooks, i.e. `useAIChatRoutes()`?